build.sbt

```diff
@@ -146,8 +147,21 @@ lazy val micrositeSettings = Seq(
 )
 
 lazy val mimaSettings = {
+  import sbtrelease.Version
+  def mimaVersion(version: String) = {
+    Version(v) match {
+      case Some(Version(major, Seq(minor, patch), _)) if patch.toInt > 0 =>
+        Some(s"${major}.${minor}.${patch.toInt - 1}")
```

Comment:
This looks right to me.  When you get to 1.0, you'll want to get a bit fancier.  I introspect tags to find the latest version in this case.  But this works great for 0.x.